src/Binovo/ElkarBackupBundle/Resources/translations/BinovoElkarBackup.en.yml

```diff
@@ -405,6 +408,15 @@ managebackups_help: |
  <p>This application depends on tahoe-lafs package. If Tahoe is not installed this box will be disabled.</p>
  <h4>Maximum parallel jobs</h4>
  The maximum parallel jobs that are allowed to be executed for this backup location.
+managerestore_help: |
+ <h4>Restore host</h4>
+ The host name or IP address of which you want to restore your backup.
+ <p>We recomend you configure a new host before start with this process.</p>
+ <h4>Source path</h4>
+ The full path in the local file system from where we would take the restore data.
+ <h4>Remote path</h4>
+ The full path on which we restore your backup remotly.
+ <p>We don't recommend you to restore over the original data.</p> 
```

Comment:
"You're advised not to restore over the original data."
I think we should explain here why (permissions and ownership)

Reply:
Perfect!